python/veles/server/proto.py

```diff
@@ -606,6 +607,12 @@ def send_msg(self, msg):
         del self.phids[msg.phid]
         self.send_msg(messages.MsgPluginHandlerUnregistered(phid=msg.phid))
 
+    async def msg_list_connections(self, msg):
+        self.send_msg(messages.MsgConnectionsReply(
+            qid=msg.qid,
+            connections=[]
```

Comment:
Why don't you return list of connections here? Do you intend to do it in separate PR together with Python clients handling for those messages?